src/v2/Apps/Fair/FairApp.tsx

```diff
@@ -79,7 +71,7 @@ const FairApp: React.FC<FairAppProps> = ({ children, fair }) => {
 
       <FairHeaderImageFragmentContainer fair={fair} />
 
-      <FairHeaderFragmentContainer fair={fair} />
+      <Spacer mt={[1, 30]} />
```

Comment:
nope, please see [available reusable units](https://github.com/artsy/palette/blob/master/packages/palette-tokens/src/themes/v3.tsx#L98-L112)

Reply:
Gotcha. Is this an example of when we should push back on the design to choose either 20 or 40px spacing? Unless the value needs to be 30px?

Reply:
According to the [figma](https://www.figma.com/file/aM3b5sodn04vMTigbQXACi/Fairs-2021?node-id=3904%3A12987) space between image and icon is `40px` but 10 of them should belong to the sticky part. Not sure if we should strongly follow those values or we can aligned it so that summary space will be `50px`? 

![Screenshot 2021-11-24 at 16 02 58](https://user-images.githubusercontent.com/44819355/143243510-5e30f0fc-33d0-4fea-a134-500d1462ba5b.png)

![Screenshot 2021-11-24 at 16 03 21](https://user-images.githubusercontent.com/44819355/143243548-c96ce3f4-ca92-47e0-877b-27a10dba1823.png)

Reply:
Ahh, okay, so it ends up being 40px in total. That makes sense. I think rolling with what you have now is fine 👍🏼